tests/pytest/test_regression.py

```diff
@@ -93,11 +93,14 @@ def run_model(rundir, model_image):
     docker_run = ['docker', 'run', '--rm']
     rundir_abs = os.path.abspath(rundir)
     rundir_mount = ['-v', f'{rundir_abs}:' + docker_runpath + '/rundir']
+    data_abs = os.path.abspath(os.path.join(rundir_abs, 'test_data'))
+    os.makedirs(data_abs)
```

Comment:
if you're running this repeatedly locally, may want?

```suggestion
    os.makedirs(data_abs, exist_ok=True)
```

Reply:
Thanks!